tensorflow/lite/micro/kernels/xtensa/xtensa_softmax.h

```diff
@@ -27,7 +27,14 @@ struct XtensaSoftmaxOpData {
   SoftmaxParams params;
   int scratch_tensor_index;
 };
-#endif  // defined(HIFI4) || defined(HIFI4_INTERNAL) || defined(HIFI5)
+#elif defined( \
+    VISION_P6)  // defined(HIFI4) || defined(HIFI4_INTERNAL) || defined(HIFI5)
```

Comment:
It looks like we use separate ifdefs elsewhere. This would probably be more readable as:

```
#endif // defined(HIFI4) || defined(HIFI4_INTERNAL) || defined(HIFI5)
#ifdef defined(VISION_P6)
```

Reply:
Updated